SYCL/README.md

```diff
@@ -1,175 +1,167 @@
 # Table of contents
  * [Overview](#overview)
- * [Execution](#execution)
- * [Main parameters](#main-parameters)
- * [LIT parameters accepted by LIT executor](#lit-parameters-accepted-by-lit-executor)
- * [LIT features which can be used to configure test execution](#lit-features-which-can-be-used-to-configure-test-execution)
+ * [Prerequisites](#prerequisites)
+ * [Build and run tests](#build-and-run-tests)
+ * [CMake parameters](#cmake-parameters)
+ * [Special test categories](#special-test-categories)
+ * [Creating or modifying tests](#creating-or-modifying-tests)
+   * [LIT feature checks](#lit-feature-checks)
+   * [llvm-lit parameters](#llvm-lit-parameters)
 
 # Overview
-This directory contains SYCL-related tests distributed in subdirectories:
- - Basic - tests used for sanity testing. Building, executing and checks are
-   defined using insource comments with LIT syntax.
- - External - contains infrastructure for running tests which sources are
-   stored outside of this repository (see [External/README.md](External/README.md) for details).
- - ExtraTests - contains infrastructure for picking up LIT style tests
-   from external directory passed in SYCL_EXTRA_TESTS_SRC. 
-
-# Execution
+This directory contains SYCL-related tests distributed in subdirectories based
+on testing scope.
+
+# Prerequisites
+
+ - DPC++ compiler. Can be built following these
+   [instructions](https://github.com/intel/llvm/blob/sycl/sycl/doc/GetStartedGuide.md#build-dpc-toolchain)
```

Comment:
Shouldn't we recommend to take pre-built binaries instead of asking to build it from sources? E.g. Contributing guide describes CI checks taking nightly builds.

Reply:
Updated text.